Add k8s and GCP recs to ccm recs docs (#30259)
DataDog/documentation: content/en/cloud_cost_management/recommendations/_index.md

      recommendation_type: Over-provisioned Kubernetes Containers
      recommendation_description: Containers with less than 30% max CPU or memory utilization.
      recommendation_prerequisites: "[Datadog profiling agent](/profiler/enabling/)"
    - category: Over-provisioned resource
      cloud_provider: AWS
      resource_type: Kubernetes Deployment
      recommendation_type: Downsize Kubernetes Deployments

>       recommendation_description: Containers use a fraction of their requested CPU or memory

Reviewer: ```suggestion
      recommendation_description: Containers are using only a fraction of their requested CPU or memory.
```

      recommendation_prerequisites: "[Datadog Agent](/agent/)"
    - category: Over-provisioned resource
      cloud_provider: Azure
      resource_type: Kubernetes Deployment
      recommendation_type: Downsize Kubernetes Deployments

>       recommendation_description: Containers use a fraction of their requested CPU or memory

Reviewer: ```suggestion
      recommendation_description: Containers are using only a fraction of their requested CPU or memory.
```

      recommendation_prerequisites: "[Datadog Agent](/agent/)"
    - category: Over-provisioned resource
      cloud_provider: GCP
      resource_type: Kubernetes Deployment
      recommendation_type: Downsize Kubernetes Deployments

>       recommendation_description: Containers use a fraction of their requested CPU or memory

Reviewer: ```suggestion
      recommendation_description: Containers are using only a fraction of their requested CPU or memory.
```

      recommendation_type: Purchase Flexible CUD for Cloud Run Job
      recommendation_description: Cloud Run Jobs that would benefit from flexible committed use discounts.
      recommendation_prerequisites: ""
    - category: Rate optimization
      cloud_provider: GCP
      resource_type: Storage Bucket
      recommendation_type: Transition Cloud Storage Bucket to Autoclass

>       recommendation_description: The objects in the storage bucket could be automatically migrated to archival tiers for better rates.

Reviewer: ```suggestion
      recommendation_description: Objects in the storage bucket can be automatically migrated to archival tiers for better rates.
```

      recommendation_prerequisites: ""
    - category: Unused resource
      cloud_provider: GCP
      resource_type: Compute Address
      recommendation_type: Delete Unused Compute IP Address

>       recommendation_description: Unused Compute IP address can be deleted.

Reviewer 2: ```suggestion
      recommendation_description: Unused compute IP address can be deleted.
```

Author: Fixed!

>       recommendation_description: Unused compute IP address can be deleted.

Reviewer: Or "An unused compute IP address can be deleted."

```suggestion
      recommendation_description: Unused compute IP addresses can be deleted.
```

      recommendation_prerequisites: ""
    - category: Unused resource
      cloud_provider: GCP
      resource_type: Compute Global Address
      recommendation_type: Delete Unused Compute Global IP Address

>       recommendation_description: Unused Compute Global IP address can be deleted.

Reviewer 2: ```suggestion
      recommendation_description: Unused compute global IP address can be deleted.
```
